operations/security_and_authentication/Change_NCN_Image_Root_Password_and_SSH_Keys_on_PIT_Node.md

```diff
@@ -10,9 +11,11 @@ There is some common preparation before making the Kubernetes image for master n
 
 ## Common Preparation
 
-1. Prepare new SSH keys on the PIT node for the root account in advance. The same key information will be added to both k8s-image and ceph-image.
+1. Prepare new SSH keys on the PIT node for the root account in advance. The same key information will be added to both `k8s-image` and `ceph-image`.
 
-   Either replace the root public and private SSH keys with your own previously generated keys or generate a new pair with `ssh-keygen(1)`. By default `ssh-keygen` will create an RSA key, but other types could be chosen and different filenames would need to be substituted in later steps.
+   Either replace the root public and private SSH keys with your own previously generated keys or generate a new pair
+   with `ssh-keygen(1)`. By default `ssh-keygen` will create an RSA key, but other types could be chosen and different
```

Comment:
While CSM users should be aware, we ostensibly only support key pairs without the use of passphrases here.  Applies to both docs in the PR.